pymongo/collection.py

```diff
@@ -673,7 +673,7 @@ def _update(session, sock_info, retryable_write):
 
     def replace_one(self, filter, replacement, upsert=False,
                     bypass_document_validation=False, collation=None,
-                    hint=None, session=None):
+                    hint=None, session=None, let=None):
```

Comment:
Can you open a new ticket to add `let` to the bulk_write models (UpdateOne, ReplaceOne, etc)? We neglected to do that in PYTHON-2957.

Reply:
https://jira.mongodb.org/browse/PYTHON-3068